pulsar-function-go/pf/context.go

```diff
@@ -57,6 +57,11 @@ func (c *FunctionContext) GetOutputTopic() string {
 	return c.instanceConf.funcDetails.GetSink().Topic
 }
 
+// user can choose outputTopic hera
+func (c *FunctionContext) SetOutputTopic(outputTopic string) {
```

Comment:
Can you follow the API convention of Java Function?

Reply:
ok,I will do it later

Reply:
@sijie can I join your discussion in how function walk，It's good for me to write code that you like